modelci/cli/modelhub.py

```diff
@@ -26,7 +26,9 @@
 from modelci.ui import model_view, model_detailed_view
 from modelci.utils import Logger
 from modelci.utils.misc import remove_dict_null
-
+from modelci.types.bo import Framework as saveFramework, Engine as saveEngine, Task as saveTask, IOShape as convertIO
```

Comment:
Why you change the import name? I think Framework, Engine, ... are fine

Reply:
''Framework'', ''Engine'',.. are already used by 

>24  from modelci.types.models import Framework, Engine, IOShape, Task, Metric, ModelUpdateSchema

Currently all convert functions are only compatible with  'saveFramework'...

Reply:
my advice is to change them to `FrameworkLagacy`, ... to indicate that we gonna remove them in future

Reply:
`saveFramework` is not clear